<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2109</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign In</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/prefix1/Img/icons8-school.png</normaloff>:/new/prefix1/Img/icons8-school.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-image:url(:/new/prefix1/Img/background.png);
	background-repeat: no-repeat;
 	 background-position: top right;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>200</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>200</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item row="0" column="2">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>661</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>661</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_2{
border-radius:10px ;
background-color: rgba(0,0,0,0.4);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>371</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame{
    border-radius:10px ;
	background-color: rgba(1, 57, 44, 0.7);
} </string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>35</number>
        </property>
        <property name="topMargin">
         <number>35</number>
        </property>
        <property name="rightMargin">
         <number>35</number>
        </property>
        <property name="bottomMargin">
         <number>35</number>
        </property>
        <item row="2" column="0">
         <widget class="QLineEdit" name="password">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Parol</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="kirish">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #ff1a1a;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #e6005c;
  margin: 3px;
  color:#FFF;
}</string>
          </property>
          <property name="text">
           <string>Kirish</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="email">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="placeholderText">
           <string>example@gmail.com</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255); </string>
          </property>
          <property name="text">
           <string>Email va parolni kiriting</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="styleSheet">
           <string notr="true">margin-bottom:30px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Parolni ko'rsatish</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="forgot">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>580</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton {
 background-color: rgba(255, 255, 255,0);
  border: none;
  color: #fff;
  text-align: center;
 font-size: 16px;
}
.QPushButton:hover{
  background-color: rgba(255, 255, 255,0);
 color:rgb(0, 255, 0);
 font-size: 16px;
} </string>
       </property>
       <property name="text">
        <string>Forgot password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="acount_icon">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
	background-color: rgb(255, 33, 73);
	border-radius:75px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/prefix1/Img/user_log.png</normaloff>:/new/prefix1/Img/user_log.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="create_acount">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>580</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton {
 background-color: rgba(255, 255, 255,0);
  border: none;
  color: #fff;
  text-align: center;
 font-size: 16px;
}
.QPushButton:hover{
  background-color: rgba(255, 255, 255,0);
 color:rgb(0, 255, 0);
 font-size: 16px;
} </string>
       </property>
       <property name="text">
        <string>Create account</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>750</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>800</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_3{
border-radius:10px ;
background-color: rgba(0,0,0,0.4);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>371</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_4{
    border-radius:10px ;
	background-color: rgba(1, 57, 44, 0.7);
} </string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>35</number>
        </property>
        <property name="topMargin">
         <number>35</number>
        </property>
        <property name="rightMargin">
         <number>35</number>
        </property>
        <property name="bottomMargin">
         <number>35</number>
        </property>
        <item row="0" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="signUp">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #ff1a1a;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #e6005c;
  margin: 3px;
  color:#FFF;
}</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255); </string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="styleSheet">
           <string notr="true">margin-bottom:30px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Parolni ko'rsatish</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="email_c">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="placeholderText">
           <string>example@gmail.com</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="password1">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Parol</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="ism">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="placeholderText">
           <string>Ism</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="fam">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="placeholderText">
           <string>Familya</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="password2">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Parol qayta kiriting</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="acount_icon_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
	background-color: rgba(255, 33, 73,0);
	border-radius:75px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/prefix1/Img/user_log.png</normaloff>:/new/prefix1/Img/user_log.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="orqaga_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>720</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton {
 background-color: rgba(255, 255, 255,0);
  border: none;
  color: #fff;
  text-align: center;
 font-size: 16px;
}
.QPushButton:hover{
  background-color: rgba(255, 255, 255,0);
 color:rgb(0, 255, 0);
 font-size: 16px;
} </string>
       </property>
       <property name="text">
        <string>Orqaga qaytish</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_5">
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_5{
border-radius:10px ;
background-color: rgba(0,0,0,0.4);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="orqaga">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>540</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">.QPushButton {
 background-color: rgba(255, 255, 255,0);
  border: none;
  color: #fff;
  text-align: center;
 font-size: 16px;
}
.QPushButton:hover{
  background-color: rgba(255, 255, 255,0);
 color:rgb(0, 255, 0);
 font-size: 16px;
} </string>
       </property>
       <property name="text">
        <string>Orqaga qaytish</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>371</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_7{
    border-radius:10px ;
	background-color: rgba(1, 57, 44, 0.7);
} </string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>35</number>
        </property>
        <property name="topMargin">
         <number>35</number>
        </property>
        <property name="rightMargin">
         <number>35</number>
        </property>
        <property name="bottomMargin">
         <number>35</number>
        </property>
        <item row="2" column="0">
         <widget class="QLineEdit" name="newparol1">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>New parol</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="styleSheet">
           <string notr="true">margin-bottom:30px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Parolni ko'rsatish</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="forgotNext">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #ff1a1a;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #e6005c;
  margin: 3px;
  color:#FFF;
}</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="newparol2">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>New parol</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255); </string>
          </property>
          <property name="text">
           <string>Emailingizni kiriting</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="email_2">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="placeholderText">
           <string>example@gmail.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="kod">
          <property name="styleSheet">
           <string notr="true">height:40px;
color: #fff;
font-size:20px;
margin-bottom:15px;
background-color: rgba(1, 57, 44, 0);
border:none;
border-bottom: 2px solid red;</string>
          </property>
          <property name="placeholderText">
           <string>Kod</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="kirishForgot">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">.QPushButton {
  border-radius: 4px;
  background-color: #ff1a1a;
  border: none;
  color: #000;
  text-align: center;
  font-size: 22px;
  padding: 10px;
  margin: 5px;
}
.QPushButton:hover{
  border-radius: 5px;
  background-color: #e6005c;
  margin: 3px;
  color:#FFF;
}</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2109</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>email</tabstop>
  <tabstop>password</tabstop>
  <tabstop>kirish</tabstop>
  <tabstop>create_acount</tabstop>
  <tabstop>forgot</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>acount_icon_2</tabstop>
  <tabstop>ism</tabstop>
  <tabstop>fam</tabstop>
  <tabstop>email_c</tabstop>
  <tabstop>password1</tabstop>
  <tabstop>password2</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>signUp</tabstop>
  <tabstop>orqaga_2</tabstop>
  <tabstop>email_2</tabstop>
  <tabstop>kod</tabstop>
  <tabstop>newparol1</tabstop>
  <tabstop>newparol2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>forgotNext</tabstop>
  <tabstop>kirishForgot</tabstop>
  <tabstop>orqaga</tabstop>
  <tabstop>acount_icon</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
